subsys/bluetooth/host/mesh/models/Kconfig

```diff
@@ -0,0 +1,37 @@
+# Kconfig - Bluetooth Mesh Models configuration options
+
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig BT_MESH_MODELS
+	bool "Bluetooth Mesh models support"
+	depends on BT_MESH
+	default n
+	help
+	  This option enables build-in Mesh models.
+
+if BT_MESH_MODELS
+
+config BT_MESH_MODEL_BATTERY_SRV
+	bool "Battery server model"
+	depends on BT_MESH_MODELS
+	default n
+	help
+	  Use this option to enable build-in battery model handling.
+
+if BT_MESH_MODEL_BATTERY_SRV
+
+config BT_MESH_MODEL_BATTERY_SRV_CNT
+	int "Number of Battery server models"
+	range 0 5
+	default 0
+	depends on BT_MESH_MODEL_BATTERY_SRV
```

Comment:
Shouldn't this tie into the battery subsystem and its information about how many batteries the system has?

Reply:
It could be, but I've not seen such battery subsystem.

Reply:
You should sync up your work together with #9735

Reply:
Updated with TODO comment to pay attention once battery subsys will be integrated.